termwiz/src/tmux_cc/mod.rs

```diff
@@ -285,8 +255,8 @@ fn parse_line(line: &str) -> anyhow::Result<Event> {
         Rule::layout_change => {
             let mut pairs = pair.into_inner();
             let window = parse_window_id(pairs.next().unwrap())?;
-            let layout = pairs.next().and_then(parse_window_layout).unwrap();
-            let visible_layout = pairs.next().and_then(parse_window_layout);
+            let layout = unvis(pairs.next().unwrap().as_str())?;
```

Comment:
Can you defang these unwraps too? I realize that a bunch of these are inherited from the existing code